### ISSUE TITLE
Incorporate a concept of “Type” or “Tier” or “Scheme” for each HEFER request

### ISSUE THREAD

**Issue by CrispinF** [exeGesIS-SDM/ELM#9]
Incorporate a concept of “Type” or “Tier” or “Scheme”, for future-proofing. This year’s cases will all be "Sustainable Farming Incentive".
In this pilot this might be shown as a non-editable drop-down, perhaps, to indicate that in future there will be options.
Note the implications for #8 and #7.

**Comment by sarahpoppy** [comment]
Yes that sounds sensible. The current language that is being used is a "component", such that ELM comprises 3 components:

Sustainable Farming Incentive
Local Nature Recovery
Landscape Recovery

**Comment by epennifold** [comment]
This is what we would do to enable this - but we think it is not MVP.

Add field on request table 'RequestType'
Add RequestTypeLUT with values
Non-editable drop-down on Request HEFER page
